controllers/persistentvolumeclaim_controller_test.go

```diff
@@ -78,7 +78,7 @@ func newPVCWithFinalizer() *corev1.PersistentVolumeClaim {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:       "sample",
 			Namespace:  "test",
-			Finalizers: []string{topolvm.PVCFinalizer, "topolvm.io/finalizer1", topolvm.PVCFinalizer, topolvm.PVCFinalizer, "topolvm.io/finalizer2", topolvm.PVCFinalizer},
+			Finalizers: []string{topolvm.GetPVCFinalizer(), "topolvm.io/finalizer1", "topolvm.io/finalizer2"},
```

Comment:
```suggestion
			Finalizers: []string{topolvm.GetPVCFinalizer(), "topolvm.io/finalizer1", topolvm.GetPVCFinalizer(), topolvm.GetPVCFinalizer(), "topolvm.io/finalizer2", topolvm.GetPVCFinalizer()},
```
There were four `topolvm.PVCFinalizer`s in the original code. This is not a bug but intentional code for boundary testing.